app/views/hyrax/base/_form_progress.html.erb

```diff
@@ -8,9 +8,13 @@
           <% if Hyrax.config.work_requires_files? %>
             <li class="incomplete" id="required-files"><%= t('.required_files') %></li>
           <% end %>
-          <% if Flipflop.show_deposit_agreement? && Flipflop.active_deposit_agreement_acceptance? %>
+<%
+=begin%>
+ <% if Flipflop.show_deposit_agreement? && Flipflop.active_deposit_agreement_acceptance? %>
```

Comment:
These `Flipflop` lines indicate a setting from the dashboard:
![image](https://github.com/OregonDigital/OD2/assets/11052958/51239886-23ef-4b4e-9c73-3541ddba559d)

Is disabling these two values enough to close this ticket as well?

Reply:
I just checked, it looks like it would be enough to close the ticket.
![Settings disabled](https://github.com/OregonDigital/OD2/assets/54194852/619f523c-2f8f-4f04-bd2e-318e96b58dd5)
![Menu without deposit agreement](https://github.com/OregonDigital/OD2/assets/54194852/f4a8e895-9515-4563-ad6f-ed7879947cb0)

Reply:
Awesome! can you change it on staging an move ticket to QA? When it passes you can go ahead and change it on prod too, no need to wait for next deploy round